src/applications/mhv-medical-records/tests/e2e/medical-records-download-functionality-vaccines.cypress.spec.js

```diff
@@ -1,8 +1,8 @@
-import moment from 'moment-timezone';
 import MedicalRecordsSite from './mr_site/MedicalRecordsSite';
 import VaccinesListPage from './pages/VaccinesListPage';
 import VaccineDetailsPage from './pages/VaccineDetailsPage';
 import defaultVaccines from './fixtures/vaccines/vaccines.json';
+import { currentDateAddSecondsCCD } from '../../util/dateHelpers';
```

Comment:
Does "CCD" in this function name refer to our CCD report? And if so, why is it being used in a test for Vaccines?

Reply:
Instead of CCD we should probably call it currentDateAddSecondsDownloadedFile